src/event-sources/aws/api-gateway-v2.js

```diff
@@ -63,7 +63,8 @@ function getResponseToApiGateway ({
   }
 
   if (headers['set-cookie']) {
-    responseToApiGateway.cookies = headers['set-cookie']
+    responseToApiGateway.cookies = Array.isArray(headers['set-cookie']) ? headers['set-cookie'] : [headers['set-cookie']]
+    delete headers['set-cookie']
```

Comment:
may be we can remove some repeated code

```javascript
if(cookies = headers['set-cookie']) {
    responseToApiGateway.cookies = Array.isArray(cookies) ? cookies: [cookies]
    delete headers['set-cookie']
}
```

Reply:
Thank you for your reviewing.
I refactored and push it. 
Can you check it again? thank you.

Reply:
thank you, looks good to me

Reply:
Please fix the failing tests